### PR TITLE
XW-1219 | update tracking

### PR DESCRIPTION
**Pull request by tomasznapieralski**
## Description
- remove curated content edit events
- remove lightbox events
- remove some random events :D
- use clickTrack mixin when possible
- update tracking for media

## Links

* http://wikia-inc.atlassian.net/browse/XW-1219

## Reviewers

@Wikia/x-wing

### CODE REVIEW

**Comment by vforge** [comment]
Plz fix tests; LGTM if done.

File: front/main/app/components/article-content.js [hunk 1]
```diff
@@ -209,6 +209,8 @@ export default Ember.Component.extend(
 					return 'portable-infobox-link';
 				} else if ($element.closest('.context-link').length) {
 					return 'context-link';
+				} else if ($element.closest('.article-image')) {
```
Comment (rogatty) [inline front/main/app/components/article-content.js]: Will it ever get to `if ($element.closest('figcaption').length)`?

**Comment by rogatty** [comment]
@tomasznapieralski can you also add the description? Why do we remove all those events?

File: front/main/app/components/article-content.js (hunk 1 above)
Comment (tomasznapieralski) [inline front/main/app/components/article-content.js]: it is triggered for linked images

**Comment by tomasznapieralski** [comment]
@rogatty updated :+1:

**Comment by vforge** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM

**Comment by hakubo** [comment]
nice cleanup

**Pull request merged**